crates/valence_boss_bar/src/components.rs

```diff
@@ -72,5 +82,13 @@ pub struct BossBarFlags {
     _pad: u8,
 }
 
-#[derive(Component)]
-pub struct BossBarViewers(pub Vec<Entity>);
\ No newline at end of file
+/// The viewers of a boss bar.
+#[derive(Component, Default)]
+pub struct BossBarViewers {
+    /// The current viewers of the boss bar. It is the list that should be
+    /// updated.
+    pub current_viewers: Vec<Entity>,
+    /// The viewers of the last tick in order to determine which viewers have
+    /// been added and removed.
+    pub last_viewers: Vec<Entity>,
+}
```

Comment:
Maybe use the same naming of other place in the project :
- viewers
- old_viewers

Reply:
I will, thanks!